runtime/compiler/CMakeLists.txt

```diff
@@ -385,6 +385,9 @@ if(OMR_OS_LINUX)
 	target_link_libraries(j9jit PRIVATE m)
 elseif(OMR_OS_WINDOWS)
 	target_sources(j9jit PRIVATE build/scripts/j9jit.def)
+elseif(OMR_OS_AIX)
+	set_property(TARGET j9jit APPEND_STRING PROPERTY
+		LINK_FLAGS " -Wl,-bE:${CMAKE_CURRENT_SOURCE_DIR}/build/scripts/j9jit.aix.exp")
```

Comment:
equivalent UMA code is https://github.com/eclipse/openj9/blob/f0510c120a0b4491ac65b76d629d2d3ddab946c5/runtime/compiler/build/toolcfg/aix-xlc/common.mk#L222